posthog/batch_exports/sql.py

```diff
@@ -7,15 +7,24 @@
         pd.distinct_id AS distinct_id,
         toString(p.id) AS person_id,
         p.properties AS properties,
-        pd.version AS version,
-        pd._timestamp AS _inserted_at
+        pd.version AS person_distinct_id_version,
+        p.version AS person_version,
+        multiIf(
+            pd.is_updated AND p.is_updated,
+            MIN(p._timestamp, pd._timestamp),
+            pd.is_updated,
+            pd._timestamp,
+            p.is_updated,
+            p._timestamp
+        ) AS _inserted_at
```

Comment:
Required as long as we use `_timestamp` to track updates. Ideally we will eventually move to reading from kafka streams directly, and then we can track this more precisely.